cashu/core/settings.py

```diff
@@ -20,6 +20,15 @@ def find_env_file():
         env.read_env(env_file, recurse=False, override=True)
     else:
         env_file = ""
+
+    # Apply override environment variables at the end
+    # This allows OVERRIDE_* env vars to override .env file settings
+    for key, value in os.environ.items():
+        if key.startswith("OVERRIDE_"):
+            # Remove OVERRIDE_ prefix and set the environment variable
+            actual_key = key[9:]  # Remove "OVERRIDE_" (9 characters)
+            os.environ[actual_key] = value
+
```

Comment:
Was this the only way you could run a mint with fees to test the wallet?

Reply:
I must admit I was hacking a lot of different things, and it's very possible that there is a simpler solution that I missed.

> Was this the only way you could run a mint with fees to test the wallet?

Now that I look at bit more at the other tests, maybe I could copy a test from `tests/mint/test_mint_fees.py` and use that instead. For example, I guess I could make a test that uses this Wallet and Ledger approach.

I''ll try that this evening, if I have time

<img width="821" height="383" alt="image" src="https://github.com/user-attachments/assets/94781de1-8808-408b-a44f-4b94248cc917" />